### PR TITLE
Pull request for TodoList App.

### PR DESCRIPTION
**Pull request by FdI96**
Kindly request a code review.

This app shows different information dynamically by using Javascript.
It has a two forms, one to add a new project/folder and other form to add a toDo.
It has a keypad to navigate through all the projects
I used ES6 new features.

The default folder contains all the toDos and the toDos are separated in each folder.
Each toDo can be removed.

### CODE REVIEW

**Review by MkrtichSargsyan — CHANGES_REQUESTED** [review]
## _Status:_ Changes Requested :recycle:

## Hello Federico Lopez Cechini 

## To Highlight :tada:

1. You have mastered git-flow. 
2. You have used ES6 syntax
3. You have descriptive git commit messages.


However, there are some changes that you need to make in order to get your project approved

- [x] Live preview is not working
- [x] You should have a default project

- [x] project should be separated into modules.
Example of a module is const Module = (()=> { const func1 = () => {}; const func2 = () => {}; return {func1, func2}; } )()

- [x] Necessary run common commands like build, development should be in package.json inside `scripts'.[screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/scripts_requirement.png)
 Here is a basic example

       "scripts": {
          "dev": "webpack --mode development",
          "build": "webpack --mode production",
          "start": "webpack-dev-server --mode development --open" // optional
        },

- [x] I could not create a new project. After clicking on the button, a white screen appears

<hr>

Please, make the REQUIRED changes and submit your request again! :grinning:
Good luck. 

> ps: . If you have any questions about my review or need help, you can message me on slack ( @mkrtichsargsyan )

File: README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+- The default folder stores all the toDos
+
+## Built With
+
+- Html, CSS, and JavaScript
+- Bootstrap
+
+## Live Demo
+
+[Live Demo Link](https://fdi96.github.io/ToDo-List/)
```
Comment (MkrtichSargsyan) [inline README.md:21]: The live link is not working

File: package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "todolist",
+  "version": "1.0.0",
+  "description": "Todo List App for Microverse",
+  "main": "webpack.config.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (MkrtichSargsyan) [inline package.json]: Necessary run common commands like build, development should be in package.json

**Review by taiwo2 — CHANGES_REQUESTED** [review]
# STATUS : CHANGES REQUIRED ♻️ 
@FdI96  Weldone for the task, However few changes is required to get your project approved
### Highlight
- [ ]  Live preview is working
- [ ] I can create a new project
- [ ] have a default project
- [ ]The user can assign a to-do list items to a project
### Improve
- [x] User can edit the details of a to-do item

 ### once specific changes made kindly make another review request

**Review by mke2111 — APPROVED** [review]
# 😄 🥇  STATUS: APPROVED  🥇 👏🏽 

## Hey Happy Coders, 👋🏽 
Congratulations, 🥂 . You got minimum requirements for approval. 👍🏽 

### **_Well Done_** 💯 

    1. The edit feature is  now implemented successfully

### You may now merge to the develop branch and remember to submit a project completion form.
### I wish you the very best in your upcoming projects.

**Cheers 🍹 and Happy coding 💻** 

<hr>

### Best regards
Slack 🆘  Roy Mukuye
Twitter 🐦  [Roy Mukuye](https://twitter.com/Roymkenya)
LinkedIn 🔗  [Roy Mukuye](https://www.linkedin.com/in/roy-mukuye/)

**Pull request merged**